javascript/createElement/js-createelement_toast-messages/js/index.js

```diff
@@ -5,9 +5,14 @@ const addButton = document.querySelector('[data-js="add-button"]');
 const clearButton = document.querySelector('[data-js="clear-button"]');
 
 addButton.addEventListener("click", () => {
+  const newToast = document.createElement("li");
```

Comment:
I think this should go outside the event listener.

Reply:
Aha! Thank you! Makes totally sense. I was wondering, why mine only added one new toast.